src/models/User.ts

```diff
@@ -0,0 +1,63 @@
+import {
+    Sequelize,
+    Model,
+    Optional,
+    DataTypes,
+} from 'sequelize';
+
+export interface UserAttributes {
+    id: string;
+    login: string;
+    password: string;
+    role: string;
+    age: number;
+    isDeleted: boolean;
```

Comment:
This is something fine. But usually we model it in a different way, you can have a field called "deletedAt" instead of a boolean. 
In that case you would have more information about is for example when the user was deleted.
The query for it wouldn't change much even as it just change to check if deletedAt is null. 

They even have a manual for that :p
https://sequelize.org/master/manual/paranoid.html